gcloud-java-dns/src/main/java/com/google/gcloud/dns/spi/DnsRpc.java

```diff
@@ -171,4 +175,8 @@ Change getChangeRequest(String zoneName, String changeRequestId, Map<Option, ?>
    */
   ListResult<Change> listChangeRequests(String zoneName, Map<Option, ?> options)
       throws DnsException;
+
+  BatchRequest enqueueListZones(BatchRequest batch, DnsBatch.Request request,
```

Comment:
As commented above I am wondering whether we should expose this method or rather hide it in `DnsImpl`.